### PR TITLE
Add missing @param and @return types to PHPDoc blocks

### PR DESCRIPTION
**Pull request by alies-dev**
Goals: help IDE and static analysis tools to help developers :)

The PR doesn't contain any known BC changes

### CODE REVIEW

File: src/BotsManager.php [hunk 1]
```diff
@@ -232,16 +232,12 @@ public function parseBotCommands(array $commands): array
...
      * Parse an array of commands and build a list.
      *
-     * @param array $commands
+     * @param list<string|class-string<\Telegram\Bot\Commands\CommandInterface>> $commands
      *
      * @return array
      */
     protected function parseCommands(array $commands): array
     {
-        if (! is_array($commands)) {
```
Comment (alies-dev) [inline src/BotsManager.php:241]: `$commands` parameter already declared as array, no need for this type check

**Comment by irazasyed** [comment]
There's some conflict. Please fix that and I'll merge it. Appreciate the improvements.

**Comment by alies-dev** [comment]
@irazasyed thanks, done ;)

**Comment by irazasyed** [comment]
Thank you 🙏🏻

**Pull request merged**